Components/Container/index.js

```diff
@@ -0,0 +1,19 @@
+import styled, { css } from "styled-components";
+
+export const Container = styled.section`
+  flex-direction: ${(props) =>
+    props.$direction === "column" ? "column" : "row"};
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-around;
```

Comment:
```suggestion
 ${(props) =>
    props.$flex === "flex" &&
    css`
      display: flex;
      flex-wrap: wrap;
      justify-content: space-around;
      flex-direction: column
    `};
```

Do you need the flex container row at this point?

Reply:
After correcting this you can send the PR to QA :))